### PR TITLE
Add Arbitrum mainnet

### PR DESCRIPTION
**Pull request by fredlacs**

### CODE REVIEW

File: _data/chains/eip155-42161.json [hunk 1]
```diff
@@ -0,0 +1,19 @@
+{
+  "name": "Arbitrum One",
+  "chainId": 42161,
+  "shortName": "arb1",
+  "chain": "ETH",
+  "network": "mainnet",
+  "networkId": 42161,
+  "nativeCurrency": {
+    "name": "Ether",
+    "symbol": "ETH",
```
Comment (ligi) [inline _data/chains/eip155-42161.json:10]: The symbol cannot be ETH. We asked the community and only Ethereum L1 should be able to have the ETH symbol. Maybe use AETH?

**Review by ligi — CHANGES_REQUESTED** [review]

File: _data/chains/eip155-42161.json (hunk 1 above)
Comment (gsteele178) [reply _data/chains/eip155-42161.json:10]: Thank you

**Review by gsteele178 — APPROVED** [review]

File: _data/chains/eip155-42161.json (hunk 1 above)
Comment (fredlacs) [reply _data/chains/eip155-42161.json:10]: I see, I'm curious as to what the rationale behind this is.
Is there an ethereum magicians thread or something of the kind I could read up on?

Comment (ligi) [reply _data/chains/eip155-42161.json:10]: I ran this poll on twitter as it is more about users than about devs: https://twitter.com/mr_ligi/status/1393135790948036610
there are some reasons in the replies.

Comment (gsteele178) [reply _data/chains/eip155-42161.json:10]: There is no rational behind any of this.... That's the beauty of it

Comment (ligi) [reply _data/chains/eip155-42161.json:10]: *?*

**Comment by gsteele178** [comment]
The eth in etherscan, I try to withdraw it through meta mask 9 times out of
10 fails...

**Comment by ligi** [comment]
@fredlacs can you change the symbol so this can be merged?

File: _data/chains/eip155-42161.json [hunk 2]
```diff
@@ -0,0 +1,19 @@
...
+    "decimals": 18
+  },
+  "rpc": [
+    "https://arb1.arbitrum.io/rpc",
+    "wss://arb1.arbitrum.io/ws"
+  ],
+  "faucets": [],
```
Comment (credfeto) [inline _data/chains/eip155-42161.json:17]: Can you add the block explorer to this too: https://explorer.arbitrum.io

File: _data/chains/eip155-42161.json (hunk 1 above)
Comment (mccordnate) [reply _data/chains/eip155-42161.json:10]: @ligi I'm not sure Arbitrum necessarily fits under the same criteria here. For a typical bridge, it definitely makes sense to restrict it to wrapped representations when denominating in ETH. However, given that Arbitrum is an L2 roll-up solution it's fair to say this is the same ETH as what you have on Ethereum. The reason being that you can never lose the ETH in your Arbitrum wallet. Even if the sequencer goes down, transactions can be posted to the L1 contract by other parties. This allows withdrawals to always be possible.

Comment (ligi) [reply _data/chains/eip155-42161.json:10]: What is if there is a bug in the Arbitrum (or any L2) contracts? I still think there is a difference between ETH and L2-ETH - but I see your point that L2-ETH is closer to ETH than other derivatives. Ideas welcome on how to convey this to users - but IMHO we cannot call it ETH.

Comment (rahul-kothari) [reply _data/chains/eip155-42161.json:10]: Chain-list uses ARETH for arbitrum

Comment (ligi) [reply _data/chains/eip155-42161.json:10]: What do you refer to as Chain-list?

**Comment by rahul-kothari** [comment]
https://chainlist.org/

*Regards,*
*Rahul Kothari*
*www.linkedin.com/in/iamrahulkothari
<http://www.linkedin.com/in/iamrahulkothari>*
*www.twitter.com/iamRahul20x* <http://www.twitter.com/iamRahul20x>

**Comment by ligi** [comment]
chainlist.org uses the data from this repo ;-)
And ARETH is the symbol for the Rinkeby Testnet of Arbitrum

**Review by ligi — APPROVED** [review]

**Comment by fredlacs** [comment]
Sorry for the slow response time @ligi 
I updated the currency symbol to `AETH` as you suggested, also went ahead and updated explorers/rpc support

**Pull request merged**